src/main/java/org/reaktivity/nukleus/tcp/internal/TcpController.java

```diff
@@ -80,69 +73,49 @@ public String name()
         String source,
         long sourceRef,
         String target,
-        long targetRef,
-        InetAddress address)
+        long targetRef)
     {
-        return route(Role.SERVER, source, sourceRef, target, targetRef, address);
+        return route(Role.SERVER, source, sourceRef, target, targetRef);
     }
 
     public CompletableFuture<Long> routeClient(
         String source,
         long sourceRef,
         String target,
-        long targetRef,
-        InetAddress address)
+        long targetRef)
     {
-        return route(Role.CLIENT, source, sourceRef, target, targetRef, address);
+        return route(Role.CLIENT, source, sourceRef, target, targetRef);
     }
 
     public CompletableFuture<Void> unrouteServer(
         String source,
         long sourceRef,
         String target,
-        long targetRef,
-        InetAddress address)
+        long targetRef)
     {
-        return unroute(Role.SERVER, source, sourceRef, target, targetRef, address);
+        return unroute(Role.SERVER, source, sourceRef, target, targetRef);
     }
 
     public CompletableFuture<Void> unrouteClient(
         String source,
         long sourceRef,
         String target,
-        long targetRef,
-        InetAddress address)
+        long targetRef)
     {
-        return unroute(Role.CLIENT, source, sourceRef, target, targetRef, address);
+        return unroute(Role.CLIENT, source, sourceRef, target, targetRef);
     }
 
     public long count(String name)
     {
         return controllerSpi.doCount(name);
     }
 
-    private Flyweight.Builder.Visitor visitRouteEx(
-        InetAddress address)
-    {
-        if (address == null)
-        {
-            return (buffer, offset, limit) -> 0;
-        }
-
-        return (buffer, offset, limit) ->
-            routeExRW.wrap(buffer, offset, limit)
-                     .address(a -> inetAddress(address, a::ipv4Address, a::ipv6Address))
-                     .build()
-                     .sizeof();
-    }
-
     private CompletableFuture<Long> route(
         Role role,
         String source,
         long sourceRef,
         String target,
-        long targetRef,
-        InetAddress address)
+        long targetRef)
```

Comment:
Now that we've removed the `address` parameter, please file a new issue to support binding by CIDR as well.

For example, `source` is `192.168.1.0/24` will find all local addresses on all devices that match the pattern and bind each of them.

This will be even more convenient than device name for portable configuration.

Reply:
Done: https://github.com/reaktivity/nukleus-tcp.java/issues/53